src/web/views/FlowBuilder/common/action.jsx

```diff
@@ -13,23 +13,22 @@ class ActionItem extends Component {
   constructor(props) {
     super(props)
 
-    this.state = { itemId: this.textToItemId(this.props.text) }
-    this.fetchItem(this.props.text)
+    const itemId = this.textToItemId(this.props.text)
+    this.state = { itemId }
+    this.fetchItem(itemId)
```

Comment:
It's usually recommended to have side-effects in componentDidMount

Reply:
Fixed that!